model/schema/allele.yaml

```diff
@@ -245,18 +399,12 @@ classes:
         object:
             range: AffectedGenomicModel
 
-  AlleleSynonymAssociation:
-    is_a: Association
-    description: >-
-      The relationship between an allele and a synonym.
+  AlleleOriginAssociationDTO:
+    is_a: AuditedObjectDTO
     slots:
-      - synonym_scope
-    slot_usage:
-      subject:
-        range: Allele
-      object:
-        range: Synonym
-
+      - allele_curie
+      - predicate_name
+      - agm_curie
```

Comment:
because this is not inheriting from an 'evidence' slot-bearing class (like a hypothetical AssociationDTO class), 'evidence_curies' needs to be added here